docs/pages/documentation/3-querying-with-graql/dml.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Data Manipulation Language
+keywords: graql, query, data manipulation language
+last_updated: April 2017
+tags: [graql]
+summary: "Data Manipulation Language"
+sidebar: documentation_sidebar
+permalink: /documentation/graql/dml.html
+folder: graql
+---
+
+# Match
+
+`match` [`<pattern> ...`](./query.html#pattern) [`(<modifier> ...)`](#modifier)
+
+A [match](#match) will find all [answers](./query.html#answer) in the knowledge base that _satisfy_ all of the given
+[patterns](#patterns). Any optional [modifiers](#modifier) are applied to the [answers](./query.html#answer) to the
+[match](#match).
+
+The [answers](./query.html#answer) to a [match](#match) can be used with either a [get](#get-query),
+[insert](#insert-query), [delete](#delete-query) or [aggregate](#aggregate-query).
+
+## Pattern
```

Comment:
I kind of found this section much harder to follow than the rest. Not sure how to clarify it better but a lot of prerequisite knowledge is needed for this section.

Reply:
I think it is still fine for reference docs. There are/will be plenty of tutorials and examples